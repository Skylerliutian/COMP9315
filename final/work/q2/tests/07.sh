./bnl 20 10 10
