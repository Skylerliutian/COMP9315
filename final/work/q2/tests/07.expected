Running: ./bnl 20 10 10

Frames:     [00] [01] [02] [03] [04] [05] [06] [07] [08] [09]
Contents:    S09  S07  S05  S01  S06  S02  S08  S03  R19  S04
PinCount:      0    0    0    0    0    0    0    0    0    0
Popularity:    2    2    1    0    1    0    2    0    1    0
Clock: 1

#requests: 220
#releases: 220
#hits    : 0
#reads   : 220
